boot/seal.go

```diff
@@ -96,7 +96,10 @@ func sealKeyToModeenv(key secboot.EncryptionKey, model *asserts.Model, modeenv *
 		return fmt.Errorf("cannot seal the encryption key: %v", err)
 	}
 
-	// TODO:UC20: store the predictable bootchains
+	installBootChainsPath := filepath.Join(dirs.SnapFDEDirUnder(InstallHostWritableDir), "boot-chains")
```

Comment:
do we need a bootChainsFileUnder(rootdir) helper?

Reply:
same

Reply:
added